pkg/registry/authentication/tokenreview/storage.go

```diff
@@ -44,6 +45,7 @@ func (r *REST) Create(ctx genericapirequest.Context, obj runtime.Object, include
 	if !ok {
 		return nil, apierrors.NewBadRequest(fmt.Sprintf("not a TokenReview: %#v", obj))
 	}
+	validation.ValidateTokenReview(tokenReview)
```

Comment:
I think I'd prefer to just check the length inline... a validation package seems overkill for a non-persisted resource.